test/general/test_policies.pass.cpp

```diff
@@ -68,7 +68,18 @@ main()
 
     // make_device_policy
     test_policy_instance(make_device_policy<class Kernel_11>(q));
+    // The next call is incompatible with SYCL 2020 library version because the constructor of sycl::queue now is explicit:
```

Comment:
I think we have to explain more clearly (with reason mentioning) like this: 
"`make_device_policy` requires a `sycl::queue` as argument. Currently, there is no implicit conversion (implicit `syc::queue` constructor by a device selector) from a device selector to a queue. The same test call with explicit queue creation we have below."